urllib3/connectionpool.py

```diff
@@ -568,25 +590,19 @@ def urlopen(self, method, url, body=None, headers=None, retries=None,
             # Close the connection. If a connection is reused on which there
             # was a Certificate error, the next request will certainly raise
             # another Certificate error.
-            if conn:
-                conn.close()
-                conn = None
+            conn = self._cleanup_conn(conn, not release_conn)
```

Comment:
It's a little weird to me that the `release_conn` argument to `cleanup_conn` is being passed a value `not release_conn`. Can we get the two arguments around the same way somehow, or at least give the argument to `cleanup_conn` a clearer name?